core/core-configuration-layer.el

```diff
@@ -1954,7 +1954,7 @@ RNAME is the name symbol of another existing layer."
 
 (defun configuration-layer//configure-packages (packages)
   "Configure all passed PACKAGES honoring the steps order."
-  (spacemacs/init-progress-bar (length packages))
+  (when dotspacemacs-loading-progress-bar (spacemacs/init-progress-bar (length packages)))
```

Comment:
can be added here instead 

https://github.com/syl20bnr/spacemacs/blob/0bbb41ec97198b19e1e44ee604caf56596506332/core/core-progress-bar.el#L35